src/pages/components/CertificateFormHeader.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { Section } from '@digicatapult/ui-component-library'
+
+import BgImageSVG from '../../assets/images/molecules-bg-repeat.svg'
+
+export default function CertificateFormHeader() {
+  return (
+    <>
+      <GreenBgDiv>
+        <WrapperDiv>
+          <Section
+            background="#e4e4e4"
+            headingGap="0ch"
+            headingLevel={3}
+            headingSize="0em"
+            height="auto"
+            padding="1ch 1ch"
+            title=""
+            width="33%"
+          >
+            <UnorderedList>
+              <ListItem>
```

Comment:
Would it be possible to replace this `<Listitem>` with Input from our UI-Component-library as per screenshot below


it will need some extra attributes e.g. rounded corners which can be passed as 


```js
<Input
  name={'some-name'}
  disabled={true}
  styles={{ border-radius: 5px }}
  ... <<< etc this is just an idea
/>
```

Reply:
Hmm .. would this be suitable, I wonder ? since the HydrogenProd + EnergyOwner + Reg are all fixed why going trough the extra pain of making 3 disabled input fields and passing the information from the child component to the parent component ?

Also, I can still use LI and make it look like an input field with the help of margins and backgrounds .. 

Last, but not least, adding a modified element in the ui comp lib .. I don't know what to say .. i am still working on the other set of input fields therefore still not sure what i want to do regarding custom input fields in the ui-lib-com list.